src/extension.ts

```diff
@@ -386,6 +391,12 @@ function getIndentationRules(): LanguageConfiguration {
   };
 }
 
+function setupClientCommandExecutionSupport(languageClient: LanguageClient): void {
+  languageClient.onRequest(ExecuteClientCommandRequest.type, async (params: ExecuteCommandParams) => {
+    return await commands.executeCommand(params.command, ...params.arguments);
+  });
+}
+
```

Comment:
As your LemMinx PR manages 2 cases:

 * call custom client command from server
 * call custom server command from client

We should support those 2 cases in this PR. I mean we should provide a new vscode command `xml.execute.workspaceCommand` like with havein vscode-java with `java.execute.workspaceCommand`:

https://github.com/redhat-developer/vscode-java/blob/6f08b0437dd7e1b8f8b0be8185d75cb33a1697d3/src/extension.ts#L220

Reply:
Agreed, i think it should be fixed now